cpp/src/structure/cugraph.cu

```diff
@@ -71,19 +71,18 @@ void cpy_column_view(const gdf_column *in, gdf_column *out) {
 }
 
 gdf_error gdf_adj_list_view(gdf_graph *graph, const gdf_column *offsets,
-                            const gdf_column *indices,
+                            const gdf_column *indices, 
```

Comment:
Just for curiosity, why did you add an additional trailing whitespace here? Is this intended?

Reply:
Not intended

Reply:
updated